src/renderers/shared/utils/Transaction.js

```diff
@@ -229,6 +236,8 @@ var Mixin = {
   },
 };
 
+export type TransactionType = typeof Transaction.Mixin;
```

Comment:
I'm not sure I understand your suggestion, do you want me to change the API such that it does:

```js
Mixin.OBSERVED_ERROR = {};
module.exports = Mixin;
```

this way the mixin, which is the transaction is exported and does the "right" thing, but at the cost of changing the API and updating all the callsites (probably only a few).

Or do you want me to export this as `Transaction` type and then change the call site to not use `require` but instead `import type`?